Sources/Amplitude/AMPStorage.m

```diff
@@ -124,19 +110,22 @@ + (NSMutableArray *)getEventsFromDisk:(NSString *)path {
 }
 
 + (NSDictionary *)JSONFromFile:(NSString *)path {
-    NSData *data = [NSData dataWithContentsOfFile:path];
-    if (data == nil) {
-        NSString *emptyData = @"{ \"batch\": []}";
-        data = [emptyData dataUsingEncoding:NSUTF8StringEncoding];
+    NSMutableString *content = [NSMutableString stringWithContentsOfFile:path encoding:NSUTF8StringEncoding error:nil];
+    if (content == nil) {
+        content = [[NSMutableString alloc] init];
```

Comment:
possible to just return an empty result here ?

Reply:
It should be in the same format, because after we call JSONFromFile, we called ` NSArray *eventsArr = [json objectForKey:@"batch"];` in  getEventsFromDisk, otherwise we need to deal with those cases differently.

Reply:
I see, that make sense. The batch key doesn't seems useful though, maybe we can re-consider if we need this key